### ISSUE TITLE
change default EML version to 2.1.1 until 2.2.0 is officially released

### ISSUE THREAD

**Issue by jeanetteclark** [ropensci/emld#31]
I know there are workarounds for this, but I would rather not have to set global options every time I spin up an R session (or change my .Rprofile) for this. For those of us (which may just be me!) testing this package regularly it would be nice to have the default version be 2.1.1 until 2.2.0 is officially out and supported by DataONE

**Comment by jeanetteclark** [comment]
thanks Carl!

**Comment by cboettig** [comment]
ah, right, this was going the other way -- now maybe a good time to reverse it and go back to 2.2.0 as the default, yes?

**Comment by amoeba** [comment]
2.2.0 should be the default on `master` after #40. Is anyone not seeing that?

**Comment by cboettig** [comment]
yup, it is, I saw the change in https://github.com/ropensci/emld/pull/40/files#diff-66a2f207a5d0d3070029a1346bada357R14 but didn't put 2 and 2 together at first
